Tasks/CondaEnvironmentV0/main.ts

```diff
@@ -1,13 +1,13 @@
 import * as path from 'path';
-import * as task from 'vsts-task-lib/task';
+import * as task from 'azure-pipelines-task-lib/task';
 import { getPlatform } from './taskutil';
 import { condaEnvironment } from './conda';
 
 (async () => {
     try {
         task.setResourcePath(path.join(__dirname, 'task.json'));
         await condaEnvironment({
-            environmentName: task.getInput('environmentName', true),
+            environmentName: task.getInput('environmentName', true)!,
```

Comment:
Should that `!` be there?

Reply:
Yes!  That's says the value returned is not-null, and suppresses the compiler error.  (and, we know the value is non-null because we're supplying `true` to `getInput`.)